log-manager/src/main/java/io/airlift/log/JsonFormatter.java

```diff
@@ -3,20 +3,27 @@
 import com.fasterxml.jackson.core.JsonFactory;
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
 import io.airlift.json.JsonCodec;
 import io.airlift.json.JsonCodecFactory;
 import io.airlift.json.ObjectMapperProvider;
 
 import java.io.IOException;
 import java.io.StringWriter;
-import java.util.logging.Formatter;
+import java.util.Map;
 import java.util.logging.LogRecord;
 
 public class JsonFormatter
-        extends Formatter
+        extends java.util.logging.Formatter
```

Comment:
Why fully qualified?

Reply:
oops, I think because the order I refactored things in, didn't notice it.